internal/analytics/client/sync/client.go

```diff
@@ -94,6 +95,7 @@ func New(cfg *config.Instance, auth *authentication.Auth) *Client {
 		UpdateTag:     p.StrP(tag),
 		UserID:        p.StrP(userID),
 		Flags:         p.StrP(dimensions.CalculateFlags()),
+		InstanceID:    p.StrP(instanceid.ID()),
```

Comment:
Since the sync client is used in `state-svc` wouldn't this override the InstanceID given to it from the async client?